src/models/RelatedLink.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by AbhishekG on 8/9/2016.
+ */
+
+class RelatedLink{
+    url;
+    description;
+    
+    constructor(){
```

Comment:
Add URL parameter to constructor.

Add default parameter description as empty string to constructor.